yml/dp02_patches.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: dp02_dc2_images
+"@id": "#dp02_dc2_images"
+description: Information about image data products in DP0.2 processing of DC2
+tables:
+- name: lsst_skymap
+  "@id": "#lsst_skymap"
+  description: Static information about the tract-and-patch skymap used for coadds
+  columns:
+  - name: lsst_tract
```

Comment:
Why have the `lsst_` prefixes?

Reply:
In the table name, I think we can dispense with it; you're right.  In the column names, it's more important to me because the other columns are ObsCore-data-model standard names.